gitpod/workspace_config

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Basic configurations for the workspace
+
+set -e
+
+# gitpod/workspace-base needs at least one file here
+touch /home/gitpod/.bashrc.d/empty
+
+# Add git aliases
+git config --global alias.co checkout
+git config --global alias.ci commit
+git config --global alias.st status
+git config --global alias.br branch
```

Comment:
Should we add those aliases? I can imagine that those will typically different for contributors (eg I use `git s` instead of `git st` for status ;))